src/main/java/co/com/bancolombia/task/GenerateStructureTask.java

```diff
@@ -14,10 +11,7 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class GenerateStructureTask extends DefaultTask {
-    private final ModuleBuilder builder = new ModuleBuilder(getProject());
-    private final Logger logger = getProject().getLogger();
-
+public class GenerateStructureTask extends CleanArchitectureDefaultTask {
```

Comment:
Maybe will be want add print help too in this task?

Reply:
I don't think so, because in this case when no params are required this will not throw error.

Reply:
it´s ok